packages/ra-core/src/auth/CanAccess.stories.tsx

```diff
@@ -109,3 +109,45 @@ export const NoAuthProvider = () => (
         </CoreAdminContext>
     </TestMemoryRouter>
 );
+
+const data = Array.from({ length: 1000 }, (_, i) => {
+    const post = {
+        id: i + 1,
+        title: `Post ${i + 1}`,
+        body: `This is the body of post ${i + 1}`,
+    };
+
+    Array.from({ length: 100 }, (_, i) => {
+        post[`property${i + 1}`] = `Another very long property ${i + 1}`;
+    });
+
+    return post;
+});
+
+export const ManyCalls = ({
```

Comment:
It would be interesting to have the react-query debugger on this story

Reply:
Done